en/mkdocs.yml

```diff
@@ -74,6 +74,11 @@ nav:
       - '': 'security-announcements/security-advisories/index.md'
       - '2024 Advisories':
         - '': 'security-announcements/security-advisories/2024/2024-advisories.md'
+        - 'WSO2-2022-1920': 'security-announcements/security-advisories/2024/WSO2-2022-1920.md'
+        - 'WSO2-2023-2589': 'security-announcements/security-advisories/2024/WSO2-2023-2589.md'
+        - 'WSO2-2023-2827': 'security-announcements/security-advisories/2024/WSO2-2023-2827.md'
+        - 'WSO2-2023-2864': 'security-announcements/security-advisories/2024/WSO2-2023-2864.md'
+        - 'CVE-2024-0392': 'security-announcements/security-advisories/2024/CVE-2024-0392.md'
```

Comment:
How do we generally decide the menu order? Why is the `CVE-2024-0392` coming here?

Reply:
This is not related to actual listing order. This is a internal mapping. you can refer the actual order in [1].

[1] [en/docs/security-announcements/security-advisories/2024/2024-advisories.md](https://github.com/wso2/docs-security/blob/84ac2655eb7626dbf703cbee236c29e9ea7ee620/en/docs/security-announcements/security-advisories/2024/2024-advisories.md)

Reply:
However, i changed the order now.